### PR TITLE
start implementing /network/options

### PR DESCRIPTION
**Pull request by ilgooz**
resolves https://github.com/tendermint/cosmos-rosetta-gateway/issues/16

### CODE REVIEW

File: rosetta/cosmos/launchpad/network.go [hunk 1]
```diff
@@ -17,8 +23,45 @@ func (l Launchpad) NetworkList(ctx context.Context, request *types.MetadataReque
...
+	Version string `json:"version"`
+}
+
+type nodeResponse struct {
+	NodeInfo nodeInfo `json:"node_info"`
+}
+
 func (l Launchpad) NetworkOptions(ctx context.Context, request *types.NetworkRequest) (*types.NetworkOptionsResponse, *types.Error) {
-	panic("implement me")
+	addr := l.endpoint + endpointNodeInfo
```
Comment (jgimeno) [inline rosetta/cosmos/launchpad/network.go:35]: Sprintf? :D

File: rosetta/cosmos/launchpad/network.go [hunk 2]
```diff
@@ -17,8 +23,45 @@ func (l Launchpad) NetworkList(ctx context.Context, request *types.MetadataReque
...
+	resp, err := l.c.Get(addr)
+	if err != nil {
+		return nil, rosetta.NewError(1, "error getting data from node")
```
Comment (jgimeno) [inline rosetta/cosmos/launchpad/network.go:38]: This is a good moment, as you did with the other to put all the errors in the same place. I like it and is easy to see which errors our app does.

File: rosetta/cosmos/launchpad/network.go [hunk 3]
```diff
@@ -17,8 +23,45 @@ func (l Launchpad) NetworkList(ctx context.Context, request *types.MetadataReque
...
+		},
+		Allow: &types.Allow{
+			OperationStatuses: []*types.OperationStatus{
+				{
+					Status:     "SUCCESS",
+					Successful: true,
+				},
+			},
+			OperationTypes: []string{
+				"Transfer",
```
Comment (jgimeno) [inline rosetta/cosmos/launchpad/network.go:60]: Interesting this, I would like to see it related to the config we put on the Network struct we created for the Server.

File: rosetta/cosmos/launchpad/network.go (hunk 2 above)
Comment (ilgooz) [reply rosetta/cosmos/launchpad/network.go]: done by 823e030

File: rosetta/cosmos/launchpad/network.go (hunk 3 above)
Comment (ilgooz) [reply rosetta/cosmos/launchpad/network.go]: done by 823e030

File: rosetta/cosmos/launchpad/network.go (hunk 1 above)
Comment (ilgooz) [reply rosetta/cosmos/launchpad/network.go]: done by 823e030

**Pull request merged**